k8s-install/README.md

```diff
@@ -1,9 +1,16 @@
 # Kubernetes self-hosted install
 
-Installation is simple:
+Installation is simple, for Kubernetes 1.5:
 
 ```
-kubectl apply -f canal.yaml
+kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/canal.yaml
+```
+
+For Kubernetes 1.6:
+
+```
+kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/1.6/canal.yaml
+kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/1.6/rbac.yaml
```

Comment:
I don't think it matters too much but should the rbac line be first. But actually, why do we have a separate RBAC line, why aren't the two files combined?